packages/cursorless-vscode/src/registerCommands.ts

```diff
@@ -8,6 +8,7 @@ import {
   TestCaseRecorder,
   showCheatsheet,
   updateDefaults,
+  tutorialCreate,
```

Comment:
I wouldn't directly import. Better to do what all the other commands do that come from engine: let the engine constructor in extension.ts pass a function / object out that gets passed into `registerCommands`. That way your function can use other things from our composition root if necessary, eg the hatTokenMap to force "air" token to be a particular token